packages/testsuite/cypress/support/verification-utils.ts

```diff
@@ -65,6 +65,21 @@ Cypress.Commands.add("verifyRemovedFromTable", (tableId, resourceName) => {
   cy.get(`${tableWrapper} td:contains("${resourceName}")`).should("not.exist");
 });
 
+Cypress.Commands.add("verifyErrorMessageIsDiplayd", (errorMessageText) => {
```

Comment:
> verifyErrorMessageIsDiplayd

`verifyErrorMessage` could be descriptive enough

Reply:
updated to `verifyErrorMessage`